Libraries/LibWeb/Credentials/CredentialsContainer.cpp

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2024, Miguel Sacristán <miguel_tete17@hotmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "CredentialsContainer.h"
+
+#include "Credential.h"
```

Comment:
Includes should use brackets (`<>`) instead of quotes. 

Files should also be formatted using `clang-format`, see: [CONTRIBUTING.md](https://github.com/LadybirdBrowser/ladybird/blob/5deb2b4cad3755c7e9251f6b23e9b1089b029aa7/CONTRIBUTING.md)

Reply:
Weird I developed it using clion and it is supposes to have clang-format working.

Maybe a different version... 

If fixed the include